include/linklist/linklist.h

```diff
@@ -15,3 +16,10 @@ void printList(Node *head);
 Node *reverseListIterative(Node *head);
 Node *reverseListRecursive(Node *head);
 bool findLoop(Node *head);
+Node *createList         (Node *, Node *);
```

Comment:
Add variable names in the argument list without them arguments are confusing. 
type name `Node` should be `listNode`. Use proper naming conventions.